src/components/Translation.tsx

```diff
@@ -1,12 +1,26 @@
 import React from "react"
-import { Trans } from "gatsby-plugin-react-i18next"
+import { useTranslation } from "gatsby-plugin-react-i18next"
+import htmr from "htmr"
 
-interface Props extends React.ComponentProps<typeof Trans> {
+import Link from "./Link"
+
+interface Props {
   id: string
 }
 
+const transform = {
+  a: Link,
```

Comment:
Is this the only element we'd want a custom component for?

Reply:
So far I haven't seen any other html element on the translation files where we would want to use a custom React component.